### ISSUE TITLE
openio command doesn't manage correctly unicode caracters

### ISSUE THREAD

**Issue by couloum** [open-io/oio-sds#1241]
Hi,

When using `openio container list` command to list containers, if one container contains an unicode character and if the output of the command is piped or redirected to a file, an error is generated and no output is written.

Exemple:
```
$ openio container list --limit 1 --marker 'Czuj'
+------------------------------------------------------------------------------------+--------+-------+
| Name                                                                               |  Bytes | Count |
+------------------------------------------------------------------------------------+--------+-------+
| Czuj+quarantine.XXX0000.XXX000000.administrative.spam@XXXX.com                     | 156933 |    20 |
| Cécile.XXXXXX+quarantine.XXX0000.XXX000000.singles.XXX000000.advertising@XXXXXX.fr |  26587 |     1 |
+------------------------------------------------------------------------------------+--------+-------+
````

The output is correct on the terminal, but the same command piped or with outpput redirected does not work:
````
$ openio container list --limit 2 --marker 'Czuj' > file
'ascii' codec can't encode character u'\xe9' in position 419: ordinal not in range(128)
$ wc -c file
0 file

$ openio container list --limit 2 --marker 'Czuj' | hexdump -C
'ascii' codec can't encode character u'\xe9' in position 419: ordinal not in range(128)
```

This makes the output of `openio` command unusable with some basic unix string manipulation commands, like grep, sed, awk, etc...

**Comment by jkasarherou** [comment]
I believe this issue is caused by the cliff python framework used for the openio CLI.
https://bugs.launchpad.net/python-cliff/+bug/1603210

It appears to have been fixed in cliff 2.0.1.

What's the version on your system?

**Comment by couloum** [comment]
I have version 1.15.0-2ubuntu2 which is the default version installed with ubuntu 16.04.

However, thanks to your answer, I just changed the output to csv and it works for me.

Thanks
